docs/srcs/comparison.md

```diff
@@ -9,984 +9,1129 @@ There are many mod managers around these days and it's may be a bit difficult to
+                Mod tracking,
+                Mod meta info like categories etc,
+                Browser for mod description,
+            </td>
+            <td id="MO1">
+                Integration no longer works since Nexus API update.
+            <td id="Vortex">
+                Account login,
+                Download with manager,
+                Mod update check,
+                Automatic mod update for premium users,
+                Mod endorsements,
+                Mod tracking,
+                Mod meta info like categories etc,
+                Browser for mod description,
+            </td>
+            <td id="NMM">
+                Account login,
+                Download with manager,
+                Mod endorsements,
+                Mod meta info like categories etc.
+            </td>
+            <td id="WB">
+                <div class="cmp-no" />
+            </td>
+        </tr>
+        <tr>
+            <td class="feature-name">Web-integration extent</td>
+            <td id="KMM">
+                <b>NexusMods</b>: Account, mod updates, mod source, downloads.
+                <br>
+                <b>LoversLab</b>: mod source.
+                <b>TESALL.RU</b>: mod source.
+            </td>
+            <td id="MO2.MO1.Vortex.NMM" colspan="4">
+                Account, mod updates, mod source, metadata query, downloads, endorsements.
+            </td>
+            <td id="WB">
+                <div class="cmp-no" />
+            </td>
+        </tr>
+        <tr>
+            <td class="feature-name">
+                Self auto-update check
+            </td>
+            <td id="KMM">
+                <div class="cmp-no" />
+            </td>
+            <td id="MO2">
+                <div class="cmp-yes" />
+            </td>
+            <td id="MO1">
+                <div class="cmp-no" />
+            </td>
+            <td id="Vortex">
+                <div class="cmp-yes" />
+            </td>
+            <td id="NMM">
+                <div class="cmp-yes" />
+            </td>
+            <td id="WB">
+                <div class="cmp-no" />
+            </td>
+        </tr>
+        <tr>
+            <td class="feature-name" align="left">Integrated download manager <sup>5</sup></td>
+            <td id="KMM">
+                <div class="cmp-yes" />
+            </td>
+            <td id="MO2">
+                <div class="cmp-yes" />
+            </td>
+            <td id="MO1">
+                <div class="cmp-no" />
+            </td>
+            <td id="Vortex">
+                <div class="cmp-yes" />
+            </td>
+            <td id="NMM">
+                <div class="cmp-yes" />
+            </td>
+            <td id="WB">
+                <div class="cmp-no" />
+            </td>
+        </tr>
+        <tr>
+            <td class="feature-name">Mod categorization</td>
+            <td id="KMM">
+                Custom tag system.
+            </td>
+      
```

Comment:
This seems to be in the wrong place or has no CSS and causes the table to widen unnecessarily.